src/widget/wtracktext.cpp

```diff
@@ -1,109 +1,20 @@
 #include "widget/wtracktext.h"
 
-#include <QDebug>
-#include <QUrl>
-
-#include "control/controlobject.h"
 #include "moc_wtracktext.cpp"
 #include "track/track.h"
-#include "util/dnd.h"
 #include "widget/wtrackmenu.h"
 
-namespace {
-constexpr WTrackMenu::Features kTrackMenuFeatures =
-        WTrackMenu::Feature::SearchRelated |
-        WTrackMenu::Feature::Playlist |
-        WTrackMenu::Feature::Crate |
-        WTrackMenu::Feature::Metadata |
-        WTrackMenu::Feature::Reset |
-        WTrackMenu::Feature::Analyze |
-        WTrackMenu::Feature::BPM |
-        WTrackMenu::Feature::Color |
-        WTrackMenu::Feature::FileBrowser |
```

Comment:
This adds now the "WTrackMenu::Feature::RemoveFromDisk"
Do we allow that for a track loaded to the deck? 
Not tested yet.

Reply:
Yes, this is allowed since #4560. The deck is stopped and the track ejected before deletion.
(when deleting from the library we don **not** check whether the track is loaded somewhere and I didn't hear of any issues, so this is rather a cosmetic/UX measure).

Reply:
Tested and works. Thank you.